drivers/sdhc/Kconfig.emmc

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config EMMC_HOST
+	bool "EMMC driver"
```

Comment:
I am ok with the Kconfig naming here, since this driver should *theoretically* be generic across any IP implementing the SDHC host spec (if I understand the driver correctly)

However, can we add something in the Kconfig description and help string making it clear this driver is currently only validated using intel's EMMC controller, so users aren't confused?

Reply:
updated with validation scope.